extensions/wikia/CreateNewWiki/templates/CreateNewWiki_CommunityBuilderModal.php

```diff
@@ -0,0 +1,21 @@
+<div class="optin-modal">
+	<div class="optin-modal__dialog-wrapper">
+		<img
+			src="https://static.wikia.nocookie.net/913679bd-e51c-4e26-a643-22314e6d9253/scale-to-width/185"
+			alt="community builder"
+		/>
+		<p class="optin-modal__copy">
+			Want to create a new community for your favorite anime or TV show? Try out the Community Builder and start building your community today!
```

Comment:
Might be worth commenting on this file that we're only supporting english for now, which is why the text is hard-coded to english.